doc/Tests_XML_Definition_and_Sample.md

```diff
@@ -33,6 +44,186 @@ Example:
 </test_definition>
 ```
 
+#### "ref" type
+
+The reference style web test case
+
+Example:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
+<test_definition>
+  <suite category="W3C/HTML5 APIs" name="tct-audio-html5-tests">
+  ...
+    <set name="Audio-ref" type="ref" ui-auto="wd">
+      <testcase component="W3C_HTML5 APIs/Media/HTML5 The audio element" execution_type="manual" id="audio_one_inside_image_not_show" priority="P3" purpose="Check that image content inside the audio element can not be shown." status="approved" type="compliance">
+        <description>
+          <test_script_entry test_script_expected_result="0">/opt/tct-audio-html5-tests/audio/w3c/audio_001-manual.htm</test_script_entry>
+          <refer_test_script_entry timeout="90">/opt/tct-audio-html5-tests/audio/w3c/audio_content-ref.htm</refer_test_script_entry>
+        </description>
+        <specs>
+          <spec>
+            <spec_assertion category="Tizen W3C API Specifications" interface="HTMLAudioElement" section="Media" specification="HTML5 The audio element" usage="true"/>
+            <spec_url>http://www.w3.org/TR/2012/WD-html5-20121025/media-elements.html#htmlmediaelement</spec_url>
+            <spec_statement/>
+          </spec>
+        </specs>
+      </testcase>
+    </set>
+  </suite>
+</test_definition>
+```
+#### "wrt" type
+
+The test contains test application, need testkit-lite to support install/uninstall test Apps by test set name. 
```

Comment:
=> The type of tests have built test application with its `name` specified in `set`, which needs testkit-lite's support to (un)install the test application.